src/ui/map.js

```diff
@@ -54,7 +54,9 @@ const defaultOptions = {
 
     trackResize: true,
 
-    renderWorldCopies: true
+    renderWorldCopies: true,
+
+    noRefreshOnExpiration: false
```

Comment:
I'm in favor of avoiding negatives in variable names when possible, as they can make conditional logic harder to read. (I may well have stole this idea/opinion from someone here, but not actually sure.) . Proposed alternative: `refreshExpiredTiles`, with a default value of `true`?